data/avro-schema/src/main/resources/avro/net/corda/data/messaging/mediator/MediatorState.avsc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "record",
+  "name": "MediatorState",
+  "namespace": "net.corda.data.messaging.mediator",
+  "doc": "Mediator state wrapper to store replayed events",
+  "fields": [
+    {
+      "name": "state",
+      "type": "bytes",
+      "doc": "The messaging library clients state."
+    },
+    {
+      "name": "outputEvents",
+      "type": {
+        "type": "array",
+        "items": "net.corda.data.messaging.mediator.MediatorReplayOutputEvents"
+      },
+      "doc": "Records output records to be sent to the message bus for each input record processed for this state."
```

Comment:
This comment is broader than this PR, but we should really settle on using either `record` or `event` to describe these, as the terminology seems to flip flop across the codebase. This will get annoying if the terms make their way into interfaces.